src/main/java/com/github/manolo8/darkbot/core/manager/StarManager.java

```diff
@@ -12,24 +12,27 @@
 import java.util.Collection;
 import java.util.Comparator;
 import java.util.List;
+import java.util.Set;
+import java.util.function.Predicate;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 public class StarManager {
+    public static final String[] HOME_MAPS = new String[]{"1-1", "2-1", "3-1"};
+    public static final String[] OUTPOST_HOME_MAPS = new String[]{"1-8", "2-8", "3-8"};
+
     private static StarManager INSTANCE;
+    private final Graph<Map, Portal> starSystem;
 
     private static int INVALID_MAP_ID = -999;
 
-    private final Graph<Map, Portal> starSystem;
-
     public StarManager() {
         INSTANCE = this;
-        String[] HOME_MAPS = new String[]{"1-1", "2-1", "3-1"};
 
         StarBuilder mapBuild = new StarBuilder();
         mapBuild.addMap(-1, I18n.get("gui.map.loading"), "?")
                 .addMap(-2, "Home Map").addPortal(0, 0, "1-1").addPortal(0, 0, "2-1").addPortal(0, 0, "3-1")
-                .addGG(-3, "GG Escort").accessOnlyBy(54, 10500, 6500, /*"1-1", "2-1", "3-1",*/ "1-8", "2-8", "3-8") // Gotta "reserve" x-1 maps for GG eternal.
+                .addGG(-3, "GG Escort").accessOnlyBy(54, 10500, 6500, /*"1-1", "2-1", "3-1",*/ OUTPOST_HOME_MAPS) // Gotta "reserve" x-1 maps for GG eternal.
```

Comment:
Commenting-out x-1 maps was more of a dirty hack, not sure that outpost home maps should really be the parameter here.

We'd need a way for multiple maps being accessed thru the same portal, possibly have on getOrCreate find all that match "as good" (because some matchers match better than others, some are type-only, some are both type and coordinate, see Zeta gate for example).